app/app.go

```diff
@@ -87,13 +89,14 @@ import (
 	"github.com/tendermint/spm/openapiconsole"
 
 	"github.com/ChihuahuaChain/chihuahua/docs"
-
 	// this line is used by starport scaffolding # stargate/app/moduleImport
 )
 
 const (
 	AccountAddressPrefix = "chihuahua"
 	Name                 = "chihuahua"
+	upgradeName          = ""
```

Comment:
In [#81](https://github.com/CosmosContracts/juno/pull/81/files) they named this "moneta-alpha" but ours doesn't have a name, so I left it blank.

Reply:
where is this used? Just to note that we will likely want to point this out when publishing our PR

Reply:
upgrade name is used to direct cosmovisor to the new binary directory at the upgrade height. i think it will also be used for upgrade handlers.